code/modules/vending/security.dm

```diff
@@ -39,3 +39,68 @@
 
 /obj/item/vending_refill/security
 	icon_state = "refill_sec"
+
+//Detective vendor
+/obj/machinery/vending/dic
+	name = "\improper DicTech"
+	desc = "A fashion and essentials vendor for the discerning detective."
+	product_ads = "Just one more question: Are you ready to look swag?; Upgrade your LA Noir threads today!;Evidence bags? Cigs? Matches? We got it all!;Get your fix of cheap cigs and burnt coffee!;Stogies here to complete that classic noir look!;Stylish apparel here! Crack your case in style!;Fedoras for her tipping pleasure.;Why not have a donut?"
+	icon_state = "det"
+	icon_deny = "det-deny"
+	req_access = list(ACCESS_FORENSICS_LOCKERS)
+	products = list(/obj/item/clothing/suit/det_suit = 1,
+					/obj/item/clothing/suit/det_suit/grey = 1,
+					/obj/item/clothing/suit/det_suit/noir = 1,
+					/obj/item/clothing/under/rank/security/detective = 1,
+					/obj/item/clothing/under/rank/security/detective/grey = 1,
+					/obj/item/clothing/accessory/waistcoat = 1,
+					/obj/item/clothing/neck/tie/red = 1,
+					/obj/item/clothing/neck/tie/black = 1,
+					/obj/item/clothing/shoes/laceup = 1,
+					/obj/item/clothing/head/fedora/det_hat = 1,
+					/obj/item/clothing/head/fedora = 1,
+					/obj/item/clothing/neck/tie/detective = 3,
+					/obj/item/clothing/gloves/color/black = 3,
+					/obj/item/pinpointer/crew = 2,					
+					/obj/item/toy/crayon/white = 2,
+					/obj/item/radio/headset/headset_sec = 2,
+					/obj/item/flashlight/seclite = 2,
+					/obj/item/taperecorder = 2,
+					/obj/item/storage/box/evidence = 2,
+					/obj/item/storage/box/matches = 5,
+					/obj/item/storage/fancy/cigarettes/cigars = 5)
+	contraband = list(/obj/item/clothing/shoes/jackboots/aerostatic = 1,
+					/obj/item/clothing/suit/det_suit/disco/aerostatic = 1,
+					/obj/item/clothing/under/rank/security/detective/disco/aerostatic = 1,
+					/obj/item/clothing/gloves/color/black/aerostatic_gloves = 1,
+					/obj/item/clothing/glasses/hud/security/sunglasses/disco = 1,
+					/obj/item/clothing/shoes/sneakers/disco = 1,
+					/obj/item/clothing/suit/det_suit/disco = 1,
+					/obj/item/clothing/under/rank/security/detective/disco = 1)
+	premium = list(	/obj/item/detective_scanner = 1,
+					/obj/item/twohanded/binoculars = 1,
+					/obj/item/camera/detective = 1,
+					/obj/item/holosign_creator/security = 1)
+
+	refill_canister = /obj/item/vending_refill/detective
+	extra_price = 100
```

Comment:
Any attempt to actually balance this price?

Reply:
What do you suggest?